src/FileSystemUtilities.js

```diff
@@ -39,6 +39,16 @@ export default class FileSystemUtilities {
     fs.writeFile(filePath, ensureEndsWithNewLine(fileContents), callback);
   }
 
+  @logger.logifyAsync()
```

Comment:
It seems like most [FileSystemUtilities have tests](https://github.com/lerna/lerna/blob/master/test/FileSystemUtilities.js); seems worthwhile to add some for these as well.